Rename - remove_participant -> user -> identifier (#17814)
Azure/azure-sdk-for-python: sdk/communication/azure-communication-chat/README.md

### Remove thread participant

Use `remove_participant` method to remove thread participant from the thread identified by threadId.

> `identifier` is the `CommunicationUserIdentifier` you created by CommunicationIdentityClient.create_user() from User Access Tokens

Reviewer: ```suggestion
`identifier` is the `CommunicationUserIdentifier` you created by CommunicationIdentityClient.create_user() from azure-communication-identity
```

Author: The User Access Token was originally present as an anchor which had been removed. Cleaned it up and changed it to communication-identity

<!-- [User Access Tokens](#user-access-tokens)  -->
and was added into this chat thread.
- Use `identifier` to specify the `CommunicationUserIdentifier` you created
```python
chat_thread_client.remove_participant(identifier=new_user)

> # # converesely you can also do the following; provided the user_id is known

Reviewer: typo

```suggestion
# # conversely you can also do the following; provided the user_id is known
```

> # from azure.communication.identity import CommunicationUserIdentifier

Reviewer: Python uses dashes
```suggestion
# from azure-communication-identity import CommunicationUserIdentifier
```
Actually, you wouldn't import from identity because that type is included in Chat right?

On line 492 CommunicationUserIdentifier is missspelled

Author: Yep, you're correct. All the identifiers are now being exposed from `chat`. I have modified the import statement

### Remove thread participant

Use `remove_participant` method to remove thread participant from the thread identified by threadId.
`identifier` is the `CommunicationUserIdentifier` you created by CommunicationIdentityClient.create_user() from `azure-communication-identity`
<!-- [User Access Tokens](#user-access-tokens)  -->
and was added into this chat thread.
- Use `identifier` to specify the `CommunicationUserIdentifier` you created
```python
chat_thread_client.remove_participant(identifier=new_user)

# # conversely you can also do the following; provided the user_id is known
# from azure.communication.chat import CommunicationUserIdentifier
# 
# user_id = 'some user id'

> # chat_thread_client.remove_participant(user=CommunicationUserIdentifier(new_user))

Reviewer: ```suggestion
# chat_thread_client.remove_participant(identifier=CommunicationUserIdentifier(new_user))
```

and was added into this chat thread.

> - Use `user` to specify the `CommunicationUserIdentifier` you created

Reviewer: ```suggestion
- Use `identifier` to specify the `CommunicationUserIdentifier` you created
```

Reviewer: Also let's rename the argument on line 481
